src/VolleyManagement.Backend/VolleyManagement.UI/Global.asax.cs

```diff
@@ -39,7 +43,9 @@ public void FixEfProviderServicesProblem()
             "StyleCopPlus.StyleCopPlusRules",
             "SP0100:AdvancedNamingRules",
             Justification = "Sergii Diachenko: This is specific naming convention.")]
+#pragma warning disable S2325 // Methods and properties that don't access instance data should be static
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)